### PR TITLE
Initialize os-update formula

### PR DESCRIPTION
**Pull request by tacerus**
New formula for managing os-update
(https://github.com/openSUSE/os-update).

### CODE REVIEW

File: os_update-formula/pillar.example [hunk 1]
```diff
@@ -0,0 +1,18 @@
+os-update:
+  # by default, the timer is left disabled
+  # to enable it, set this to true
+  enable: false
+
+  # by default, no timer override will be installed, and the packaged default will apply
+  # to install an override, use the syntax described in systemd.timer(5) - example:
+  # timer: 'Thu *-*-01/4 02:00:00'
+  timer: false
```
Comment (cboltz) [inline os_update-formula/pillar.example:9]: `timer: false` might be misunderstood as "do not enable os-update.timer".

Maybe use `time:` instead?

Comment (tacerus) [reply os_update-formula/pillar.example:9]: Good idea.

Comment (tacerus) [reply os_update-formula/pillar.example]: Changed.

File: os_update-formula/pillar.example [hunk 2]
```diff
@@ -0,0 +1,18 @@
+os-update:
+  # by default, the timer is left disabled
+  # to enable it, set this to true
+  enable: false
```
Comment (cboltz) [inline os_update-formula/pillar.example:4]: The default in map.jinja disagrees with what the comment says ;-)

Enabling the timer by default is a good idea IMHO (why else would you include this formula?), so please update pillar.example.

Comment (tacerus) [reply os_update-formula/pillar.example]: Wonder how that happened - thanks, corrected.

**Pull request merged**